spec/features/result_filters/location_spec.rb

```diff
@@ -23,110 +24,81 @@
   describe "Selecting an option" do
     before { filter_page.load }
 
-    it "Allows the user to select across england" do
-      filter_page.across_england.click
-      filter_page.find_courses.click
+    it "Allows the user to select 'by postcode, town or city'" do
+      location_filter_results_page = PageObjects::Page::ResultFilters::LocationFilterResults.new
```

Comment:
See commit message for explanation - https://github.com/DFE-Digital/find-teacher-training/pull/62/commits/ffc7913b522c9a0e9fb901c0bd65b6d9ebf734ee

This took ages to debug :-(